docs/book/src/reference/jobs.md

```diff
@@ -60,12 +66,17 @@ Prow Postsubmits:
 Prow Periodics:
 * [periodic-cluster-api-test-main] `./scripts/ci-test.sh`
 * [periodic-cluster-api-test-mink8s-main] `./scripts/ci-test.sh`
-  * KUBEBUILDER_ENVTEST_KUBERNETES_VERSION: `1.23.5`
+  * KUBEBUILDER_ENVTEST_KUBERNETES_VERSION: `1.24.2`
 * [periodic-cluster-api-e2e-main] `./scripts/ci-e2e.sh`
   * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]|[IPv6]`
 * [periodic-cluster-api-e2e-mink8s-main] `./scripts/ci-e2e.sh`
   * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]|[IPv6]`
-  * KUBERNETES_VERSION_MANAGEMENT: `stable-1.23`
+  * KUBERNETES_VERSION_MANAGEMENT: `stable-1.24`
+* [periodic-cluster-api-e2e-dualstack-ipv6-main] `./scripts/ci-e2e.sh`
+  * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]`
+* [periodic-cluster-api-e2e-ipv6-main] `./scripts/ci-e2e.sh`
+  * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]`
+  * GINKGO_FOCUS: `[IPv6] [PR-Informing]`, IP_FAMILY: `IPv6`
```

Comment:
If we are mentioning `IP_FAMILY` then let's also mention `DOCKER_IN_DOCKER_IPV6_ENABLED` or let's drop `IP_FAMILY`. I would suggest to drop `IP_FAMILY` and probably better to focus only on `GINKGO_SKIP` and `GINKGO_FOCUS` in this page.
Thoughts?

Reply:
Let's mention all of them for now

Reply:
We'll probably simplify this in a few weeks once we decided how to run the duslstack tests long term. Then we can clean up

Reply:
Added `DOCKER_IN_DOCKER_IPV6_ENABLED` to both `ipv6` & `dualstack` periodic/presubmit jobs